spec/requests/authentication_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe 'POST /api/v1/users/sign_in', type: :request do
+  let(:user) { User.create!(email: 'foo@bar.com', password: '123123') }
+  let(:url) { '/api/v1/users/sign_in' }
```

Comment:
```suggestion
  let(:url) { user_session_path }
```
Используй рельсовый UrlBuilder (проверять путь можно в `rake routes`)

Reply:
Так же и в остальных местах, где урлы

Reply:
но на апдейт же урл_метод не прокатит, там же айдишник поста нужен...и придется просто строкой путь писать. Нет?

Reply:
`api_v1_post_path(id)`